src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java

```diff
@@ -493,7 +493,16 @@ private boolean findSourceCodeSVN(HttpMessage originalMessage) throws Exception
 								}
 							}
 							catch (Exception e) {
-								log.error ("Error executing SQL on temporary SVN SQLite database '"+ sqliteConnectionUrl + "': "+ e);
+								StringBuilder errorSb=new StringBuilder(300);
+								errorSb.append("Error executing SQL on temporary SVN SQLite database '");
+								errorSb.append(sqliteConnectionUrl);
+								errorSb.append("'");
+								if (log.isDebugEnabled()) {
```

Comment:
Sure, I understand that too. What I was saying is that the statement log.isDebugEnabled() is not needed (well, it could be added around the whole construction of the log message instead), the method debug only logs the message when isDebugEnabled() returns true, so, having that if statement or not is the same.